docs/docs/sampler.ipynb

```diff
@@ -514,14 +514,6 @@
     "# generate samples\n",
     "samples, sampler_state = sampler.sample(log_pdf, pars, state=sampler_state, chain_length=100)"
    ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "410531e1-3f36-4b22-ab3c-321b71b3532a",
-   "metadata": {},
-   "outputs": [],
-   "source": []
```

Comment:
This just removes an empty cell, right? Does `samplers.ipynb` need to be changed in this PR at all?

Reply:
I mistakenly merged `samplers.ipynb` in the previous PR, but it was not yet linked in the docs.
I had a few changes to samplers.ipynb, but most was already in the previous PR.